lib/appraisal/customize.rb

```diff
@@ -3,31 +3,33 @@
 module Appraisal
   class Customize
     def initialize(heading: nil, single_quotes: false)
-      @@heading = heading&.chomp
+      @@heading = !heading.nil? && heading.chomp
       @@single_quotes = single_quotes
     end
 
-    def self.heading(gemfile)
+    def self.heading(gemfile = nil)
```

Comment:
The RSPec tests for this method were broken, and marked as pending because this was a bug!